src/main/scala/karazin/scala/users/group/week1/homework/Homework.scala

```diff
@@ -37,30 +37,84 @@ import scala.annotation.tailrec
  */
 
 object Homework :
-
+  //===========================================================================
   object `Boolean Operators` :
 
-    def not(b: Boolean): Boolean = ???
+    def not(b: Boolean): Boolean =
+      if (b)

```

Comment:
It's not an issue but you can get rid of parenthesis: if b then false else true.